friends/src/App.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from "react";
+import "./App.css";
+import axios from "axios";
+import { Route } from "react-router-dom";
+
+import FriendsList from "./components/FriendsList";
+import FriendsForm from "./components/FriendsForm";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: []
+    };
+  }
+
```

Comment:
make sure when you start practicing the todo lists you keep using axios so that when you get to build week it will help you, also when you get to back end you will be able to create a front end faster if you have been practicing